maintainers/maintainer-list.nix

```diff
@@ -7955,6 +7955,12 @@
     githubId = 2157287;
     name = "sohalt";
   };
+  solene = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add solene``.